README.rst

```diff
@@ -16,60 +16,99 @@ between a set of devices owned by a same account.
 API
 ===
 
-* `API Design proposal
-  <https://github.com/mozilla-services/readinglist/wiki/API-Design-proposal>`_
 * `Online documentation <http://readinglist.readthedocs.org/en/latest/>`_
 
 
 Run locally
 ===========
 
-By default, readinglist persists its records inside a `Redis
-<http://redis.io/>`_  database, so it has to be installed first (see the
-"Install Redis" section below for more on this).
+*Reading List* is based on top of the `cliquet <https://cliquet.rtfd.org>`_ project, and
+as such, please refer to cliquet's documentation for more details.
 
-Once Redis is installed:
+
+For development
+---------------
+
+By default, *Reading List* persists the records and internal cache in a PostgreSQL
+database.
+
+The default configuration will connect to the `postgres` database on
+`localhost:5432`, with user/password `postgres/postgres`. See more details
```

Comment:
I'm not an .rst guy, but should these be `*postgres*` or use double backticks? They look like regular text in the rendered file in GitHub, but maybe GitHub just has bad .rst rendering (since it looks like it renders fine at http://rst.ninjs.org/)

![readinglist_readme_rst_at_162_postgres_setup_ _mozilla-services_readinglist](https://cloud.githubusercontent.com/assets/557895/6701120/7b51b7f2-ccd5-11e4-8699-76bab1611b95.png)
**Figure 1:** Note that "postgres" looks the same as the rest of the text and isn't `postgres` or *postgres*.